### PR TITLE
Added tests for map

### PR DESCRIPTION
**Pull request by yhordii**

### CODE REVIEW

File: src/test/java/com/qaprosoft/carina/demo/MyMobileTest.java [hunk 1]
```diff
@@ -6,13 +6,18 @@
...
 import com.zebrunner.agent.core.annotation.TestLabel;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 import org.testng.asserts.SoftAssert;
 
 public class MyMobileTest implements IAbstractTest, IMobileUtils {
+
+
+
+
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/MyMobileTest.java]: 3 unnecessary spaces

File: src/test/java/com/qaprosoft/carina/demo/MyMobileTest.java [hunk 2]
```diff
@@ -52,6 +57,13 @@ public void LoginTest() throws Exception {
         CarinaDescriptionPageBase carinaPageBase = loginPageBase.clickLoginBtn();
         softAssert.assertTrue(carinaPageBase.isPageOpened(), "Can't log in.");
 
+        //7th step
+        MapsPageBase mapsPageBase = carinaPageBase.navigateToMapPage();
+        Assert.assertTrue(mapsPageBase.isPageOpened(), "Can't open Map Page.");
+        Assert.assertTrue(mapsPageBase.isZoomInPresented(), "No Zoom In button on map.");
+        Assert.assertTrue(mapsPageBase.isZoomOutPresented(), "No Zoom Out button on map.");
+        Assert.assertTrue(mapsPageBase.areBtnsRightOrder(), "Button aren't in a right order.");
+
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/MyMobileTest.java]: create new test method for this task

**Comment by vrudyk** [comment]
ok

**Comment by vrudyk** [comment]
ok

File: src/test/java/com/qaprosoft/carina/demo/MyMobileTest.java [hunk 3]
```diff
@@ -62,7 +62,7 @@ public void loginTest() {
     @Test(description = "My mobile test for map")
     @MethodOwner(owner = "qpsdemo")
     @TestLabel(name = "feature", value = {"mobile", "regression"})
-    public void mapTest() {
+    public void mapTest() throws InterruptedException {
```
Comment (yhordii) [inline src/test/java/com/qaprosoft/carina/demo/MyMobileTest.java:65]: added for custom waiter yesterday, removed now

**Comment by vrudyk** [comment]
ok

**Pull request merged**